docs/core/diagnostics/eventpipe.md

```diff
@@ -0,0 +1,82 @@
+
+You can use EventPipe to trace your .NET application in many ways:
+
+* Use one of the [diagnostics tools](#tools-using-eventpipe) that are built on top of EventPipe.
+
+* Use [Microsoft.Diagnostics.NETCore.Client](https://github.com/dotnet/diagnostics/blob/master/documentation/diagnostics-client-library-instructions.md) library to write your own tool to configure and start EventPipe sessions yourself.
+
+* Use [environment variables](#trace-using-environment-variables) to start EventPipe.
+
+After you've produced a `nettrace` file that contains your EventPipe events, this can be viewed in [`PerfView`](https://github.com/Microsoft/perfview#perfview-overview) or Visual Studio.
+
+You can also analyze EventPipe traces programmatically with [TraceEvent](https://github.com/Microsoft/perfview/blob/master/documentation/TraceEvent/TraceEventLibrary.md).
+
+### Tools using EventPipe
+
+This is the easiest way to use EventPipe to trace your application. To learn more about how to use each of these tools, refer to each tools' documentation.
+
+* [dotnet-counters](./dotnet-counters.md) lets you monitor/collect various metrics emitted by the .NET runtime and core libraries, as well as custom metrics you can write.
+
+* [dotnet-gcdump](./dotnet-gcdump.md) lets you collect GC heap dump of applications for analyzing your application's managed heap.
+
+* [dotnet-trace](./dotnet-trace.md) lets you collect traces of applications to analyze for performance.
+
+## Trace using environment variables
+
+The preferred mechanism for using EventPipe is to use `dotnet-trace` or `Microsoft.Diagnostics.NETCore.Client` library.
+
+However, you can use the following environment variables to set up an EventPipe session on an app and have it write the trace directly to a file. To stop tracing, you need to exit the application.
+
+* `COMPlus_EnableEventPipe`: Setting this to `1` starts an EventPipe session that writes directly to a file. By default this is `0`.
+
+* `COMPlus_EventPipeOutputPath`: This is the path to the output EventPipe trace file when it is configured to run via `COMPlus_EnableEventPipe`. By default this is `trace.nettrace` which will be created in the same directory the app that is running from.
+
+* `COMPlus_CircularBufferMB`: This is the size of the internal buffer size that is used by EventPipe when it is configured to run via `COMPlus_EnableEventPipe`.
+
+* `COMPlus_EventPipeConfig`: This environment variable can be used to set up the EventPipe session configuration when setting up an EventPipe session with `COMPlus_EnableEventPipe`.
+
+The syntax is as following:
+
+* `<provider>:<keyword>:<level>`
+
+You can also specify multiple providers by concatenating them with comma:
+
+* `<provider1>:<keyword1>:<level1>,<provider2>:<keyword2>:<level2>`
+
+If this environment variable is not set but EventPipe is enabled by COMPlus_EnableEventPipe, it will start tracing by enabling the following providers with the following keywords and level:
```

Comment:
```suggestion
  If this environment variable is not set but EventPipe is enabled by `COMPlus_EnableEventPipe`, it will start tracing by enabling the following providers with the following keywords and levels:
```